Source/Order.swift

```diff
@@ -159,4 +154,81 @@ Represents an order placed by the user to be tracked using `ButtonMerchant.track
             case advertisingId = "advertising_id"
         }
     }
+
+    /**
+     Represents a line item in the order.
+     */
+    @objcMembers
+    @objc(LineItem) final public class LineItem: NSObject, Codable {
+
+        /**
+         The unique identifier for this line item,
+         within the scope of this order. (required).
+         */
+        let identifier: String
+
+        /**
+         The total price of all items bought in a particular line item (required).
+         (e.g. if 3 bananas were purchased for $3.00 each, total would be 900).
+         */
+        let total: Int64
+
+        /**
+         The number of unique units represented by this line item.
+         */
+        public var quantity: Int
```

Comment:
Are we going with a default value or excluding this field?

Reply:
default provided on init